tracer/src/Datadog.Trace.Coverage.collector/ICollectorLogger.cs

```diff
@@ -0,0 +1,23 @@
+// <copyright file="ICollectorLogger.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+
+#pragma warning disable SA1300
```

Comment:
```suggestion
// Disable SA1300 Warning due the `collector` lowercase in the namespace
#pragma warning disable SA1300
```

Reply:
done in [8544fbd](https://github.com/DataDog/dd-trace-dotnet/pull/2739/commits/8544fbd8c6c5425101db2dbf9ae5de5fc4f8aad0)